docs/_functions/global/DU.md

```diff
@@ -0,0 +1,31 @@
+---
+name: DU
```

Comment:
Pretty cool addition!  I can see how this would be useful.  I would like to suggest a different name, however.  How about `ADD_TO()` or `D_ADD()` ?

Reply:
I'd go for a function name starting with `D` because 1) it's doing domain stuff, 2) it works pretty like `D()` does. But I think `D_ADD()` might be a bit too misleading, as it's not "adding a domain" (what I might think of based on the name). Usually I thought about `DU()` standing for "**D**omain **U**pdate". As alternatives: What about `D_UPD()` or `D_UPDATE()` instead?

Reply:
Yes, I see your point.  It should start with `D`.  My issue with `DU` is that it is vague.  (of course, so is `D`).   I like `D_UPDATE` but I feel like that could be misinterpreted as "replace" (UPDATE in SQL is "replace if exists")

Just brainstorming here but ...

* `D_INSERT()`
* `D_MORE()`
* `D_CONTINUE()`

Reply:
I like D_MORE() as it's clear what it does, that it doesn't replace D(), and that it's not performing an SQL-style UPDATE. (D_INSERT() sounds as if it could mean "insert domain", same objection as D_ADD(). D_CONTINUE is OK, but without being able to clearly articulate why it feels weird to me.

Reply:
Consider using REC() since it is quick to parse as a human when reading down through a config.  Or consider rewriting D() to handle the updates.

Reply:
I also like `D_MORE()` actually. But @pearcec's idea is pretty good as well: Allowing `D()` to update. But not sure however if this could lead to confusion or unintended side effects.

Reply:
What about: `DE()` or `DE_EXT()` or `D_EXTEND()`?